react/src/components/UserDropdownMenu.tsx

```diff
@@ -181,45 +180,42 @@ const UserDropdownMenu: React.FC<{
           maxWidth: 300,
         }}
         placement="bottomRight"
-        open={open}
-        onOpenChange={(open) => {
-          setOpen(open);
-        }}
       >
-        {buttonRender(
-          <Button
-            type="text"
-            data-testid="user-dropdown-button"
-            // loading={isPendingInitializeSettingModal}
-            onClick={(e) => setOpen(!open)}
-            style={{
-              textOverflow: 'ellipsis',
-              whiteSpace: 'nowrap',
-              display: 'flex',
-              alignItems: 'center',
-              justifyContent: 'center',
-              marginTop: -2,
-              fontSize: token.fontSizeLG,
-            }}
-            // icon={<UserOutlined />}
-            icon={
-              <Avatar
-                size={17}
-                icon={
-                  <UserOutlined
-                    style={{ fontSize: 10, color: token.colorPrimary }}
-                  />
-                }
-                style={{
-                  // border: 1,
-                  backgroundColor: token.colorBgBase,
-                }}
-              ></Avatar>
-            }
-          >
-            {screens.lg && _.truncate(userInfo.username, { length: 30 })}
-          </Button>,
-        )}
+        {/* Add a `div` to fix the Dropdown bug when the child is a `ReverseThemeProvider`. It might be related to the Ant Design version. */}
```

Comment:
I think It would good to move this comment to Github issues for tracking. one of another reason is that we couldn't pretty sure that it is related to the ant-d problem as you mentioned in the comment

Reply:
Thank you for the suggestion. I moved the `div` from `UserDropdownMenu` to `WebUIHeader` and changed the comment because it is more related to `ReverseThemeProvider` in DropDown (ConfigProvider).